sidechain/storage/src/test_db_fixture.rs

```diff
@@ -0,0 +1,51 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::storage::SidechainStorage;
+use its_primitives::types::SignedBlock as SignedSidechainBlock;
+use rocksdb::{Options, DB};
+use std::{path::PathBuf, vec::Vec};
+
+/// Test fixture for a DB, cleans itself up when it goes out of scope.
+///
+/// Use with `_dbFixture`, not `_` - because the latter immediately drops!
+pub struct TestDbFixture {
```

Comment:
This setup is not very stable. I once had a panic in a test, and it corrupted the state because such that I had to manually delete the directories to get the tests running again.

* Now that we update the tests anyhow, could you solve: https://github.com/integritee-network/worker/issues/401